nsot/middleware/auth.py

```diff
@@ -50,3 +51,31 @@ def configure_user(self, user):
 
         log.debug('Created new user: %s', user)
         return user
+
+
+class NsotObjectPermissionsBackend(ObjectPermissionBackend):
+    """Custom backend that overloads django-guardian's has_perm method."""
+    def has_perm(self, user_obj, perm, obj=None):
+        """
+        Returns ``True`` if ``grp_obj`` has ``perm`` for ``obj``. If no
```

Comment:
Docstring is still referencing `grp_obj` vs `user_obj`

Reply:
Oops, guess I forgot to change this. Done now.